contributors/devel/development.md

```diff
@@ -284,17 +284,31 @@ make cross
 ```sh
 cd $working_dir/kubernetes
 
+# Run all hack/update scripts
+# For faster deployment, run only relevant scripts:
+# hack/update-gofmt.sh (when adding new files)
+# hack/update-bazel.sh (when updating dependencies / imports)
+hack/update-all.sh
```

Comment:
noted, i will make the change accordingly. 
should i retain examples such as `hack/update-gofmt.sh` for specific scripts?